### PR TITLE
fix: underline doesn't show up initially

### LINKED ISSUE #756: ScrollableTabView initialPage={0} doesn't display the underline highlight
Only happens if I use `ScrollableTabBar` on `renderTab` rather than `DefaultTabBar`

### PR DESCRIPTION
**Pull request by bksantiago**
Prevent an undefined value for `offset.value`

closes #756

### CODE REVIEW

File: ScrollableTabBar.js [hunk 1]
```diff
@@ -59,6 +59,9 @@ const ScrollableTabBar = React.createClass({
   },
 
   updateView(offset) {
+    if (offset.value === undefined) {
```
Comment (syzer) [inline ScrollableTabBar.js]: This one is better
```js
if (!offset.value) {
```

Comment (bksantiago) [reply ScrollableTabBar.js]: Applied in c595108c71bf864e2cfa047affe3883d74fa49d9

**Comment by syzer** [comment]
@skv-headless WDYT?

**Comment by syzer** [comment]
I see conflicts.. maybe rebase or resolve with merge?

**Pull request closed without merging**